lib/install/validate/validator.go

```diff
@@ -205,6 +208,15 @@ func (v *Validator) basics(ctx context.Context, input *data.Data, conf *config.V
 	conf.SetName(input.DisplayName)
 	conf.SetDebug(input.Debug.Debug)
 	conf.Name = input.DisplayName
+
+	scratchSize, err := units.FromHumanSize(input.ScratchSize)
+	if err != nil { // TODO set minimum size of scratch disk
+		v.NoteIssue(errors.Errorf("Invalid default image size %s provided", input.ScratchSize))
+	} else {
+		conf.ScratchSize = scratchSize / bytesToKilobytes
```

Comment:
Ha thanks for this, I also made this change for unit handling in `VolumeCreate` 👍